### PR TITLE
Revise workflow terms

### PR DESCRIPTION
**Pull request by OR13**
aligns with: https://github.com/w3c-ccg/traceability-interop/pull/70

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -285,84 +285,140 @@ <h2>Terminology</h2>
...
-                  "https://www.w3.org/2018/credentials/examples/v1"
-                ],
-                "type": ["VerifiableCredential"],
-                "issuer": "did:example:Wz4eUg7SetGfaUVCn8U9d62oDYrUJLuUtcy619",
-                "correlationId": "d2dfc100-9384-49e4-b547-5e7d84ec5bf5",
-                "credentialSubject": {
-                  "originator": "Some User or System",
-                  "product": "Blueberries",
-                }...
+                "correlationId": "d2dfc100-9384-49e4-b547-5e7d84ec5bf5"
```
Comment (OR13) [inline docs/index.html:307]: goal here to make it clear this is an identifier used to group things... and describe the cases for grouping in the following sections.

- groups of definitions
- groups of instances
- groups of nodes in a path

File: docs/index.html [hunk 2]
```diff
@@ -285,84 +285,140 @@ <h2>Terminology</h2>
...
+          <dfn data-lt="WorkflowDefinition">Workflow Definition</dfn>
+        </dt>
+        <dd>
+          <p>
+            The sequence of industrial, administrative, or other processes through which a 
+            piece of work passes from initiation to completion.
+          </p>
+          <p>
+            In the context of this api profile, we consider the specifics of workflows out so scope.
+            We acknowledge that different use cases, may have complicated workflows, which may yield
```
Comment (TallTed) [inline docs/index.html:322]:
```suggestion
            In the context of this API profile, we consider the specifics of workflows out of scope.
            We acknowledge that different use cases may have complicated workflows, which may yield 
```

File: docs/index.html [hunk 3]
```diff
@@ -285,84 +285,140 @@ <h2>Terminology</h2>
...
+            }
+            </pre>
+        </dd>
+        <dt>
+          <dfn data-lt="PATH|PATHs">PATH</dfn>
+        </dt>
+        <dd>
+          <p>
+            A distinct path for a distinct set of items being traced that either is declared
+            in advance by a user or system, or a root path that has already occurred
```
Comment (TallTed) [inline docs/index.html:397]:
```suggestion
            in advance by a user or system, or is a root path that has already occurred
```

File: docs/index.html [hunk 4]
```diff
@@ -285,84 +285,140 @@ <h2>Terminology</h2>
...
+            point (a node) to the end destination.  Any change in state, location, or 
+            control (legal or physical responsibility of some kind for the goods) would be
+            described as a node represented by a VP which has the PATH CORRELATION ID
+            attached to it. 
+           </p>
+
+           <p>
+             A path represents a strict subset of the information available to an observer 
+             of all "Workflow Instance" related credentials. 
+             Meaning that PATHs represent partial information related to a workflow instance.
```
Comment (TallTed) [inline docs/index.html:418]:
```suggestion
             of all "Workflow Instance" related credentials. In other words,
             PATHs represent partial information related to a workflow instance.
```

**Review by mprorock — COMMENTED** [review]
I like the examples here - I think we need to keep the concept of "paths" though as a general term - highly useful and clarifies the graph like structure of the data we are dealing with.

**Comment by OR13** [comment]
@mprorock I agree.. I think path is needed to explain that not all parties will see the "full graph" but that some parties may require a partial view of that graph that aligns with a particular path.

File: docs/index.html [hunk 5]
```diff
@@ -285,84 +285,140 @@ <h2>Terminology</h2>
...
-          PATH CORRELATION ID.
-
-          In a real world context, a PATH might be created to describe one party
-          requesting a set of goods from some other location.  The PATH can then
-          be used to group and associate the nodes as those goods move from the origination
-          point (a node) to the end destination.  Any change in state, location, or 
-          control (legal or physical responsibility of some kind for the goods) would be
-          described as a node represented by a VC which has the PATH CORRELATION ID
-          attached to it. 
-        </dd>
```
Comment (mprorock) [inline docs/index.html:309]: let's keep this, and add a definition for WORKFLOW

**Review by mprorock — CHANGES_REQUESTED** [review]
let's keep path, and add a very specific definition for WORKFLOW

**Review by mkhraisha — APPROVED** [review]
on further review this lgtm

File: docs/index.html (hunk 5 above)
Comment (mprorock) [reply docs/index.html:309]: perfect - i am dumb and misread.

**Review by mprorock — APPROVED** [review]

**Pull request merged**